ShestakUI/Modules/Skins/Blizzard/EncounterJournal.lua

```diff
@@ -111,6 +111,13 @@ local function LoadSkin()
 	T.SkinScrollBar(EncounterJournalEncounterFrameInstanceFrameLoreScrollFrameScrollBar)
 	T.SkinScrollBar(EncounterJournalEncounterFrameInfoOverviewScrollFrameScrollBar)
 	T.SkinScrollBar(EncounterJournalEncounterFrameInfoBossesScrollFrameScrollBar)
+	
+	EncounterJournalSuggestFrame.Suggestion1.reward:StripTextures()
+	EncounterJournalSuggestFrame.Suggestion1.reward:SetTemplate("Transparent")
```

Comment:
Try add
EncounterJournalSuggestFrame.Suggestion1.reward.icon:SetMask("")
EncounterJournalSuggestFrame.Suggestion1.reward.icon:SetTexCoord(0.1, 0.9, 0.1, 0.9)
It will turn from circle to square texture